2025-03-09 22:49:18,279 INFO    MainThread:28736 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-09 22:49:18,280 INFO    MainThread:28736 [wandb_setup.py:_flush():67] Configure stats pid to 28736
2025-03-09 22:49:18,280 INFO    MainThread:28736 [wandb_setup.py:_flush():67] Loading settings from C:\Users\singh\.config\wandb\settings
2025-03-09 22:49:18,280 INFO    MainThread:28736 [wandb_setup.py:_flush():67] Loading settings from C:\Users\singh\OneDrive\Documents\GitHub\DA6401_Assignment-01\wandb\settings
2025-03-09 22:49:18,280 INFO    MainThread:28736 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-09 22:49:18,281 INFO    MainThread:28736 [wandb_init.py:setup_run_log_directory():647] Logging user logs to C:\Users\singh\OneDrive\Documents\GitHub\DA6401_Assignment-01\wandb\run-20250309_224918-Question-1\logs\debug.log
2025-03-09 22:49:18,282 INFO    MainThread:28736 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to C:\Users\singh\OneDrive\Documents\GitHub\DA6401_Assignment-01\wandb\run-20250309_224918-Question-1\logs\debug-internal.log
2025-03-09 22:49:18,282 INFO    MainThread:28736 [wandb_init.py:init():761] calling init triggers
2025-03-09 22:49:18,282 INFO    MainThread:28736 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-09 22:49:18,282 INFO    MainThread:28736 [wandb_init.py:init():784] starting backend
2025-03-09 22:49:18,282 INFO    MainThread:28736 [wandb_init.py:init():788] sending inform_init request
2025-03-09 22:49:18,315 INFO    MainThread:28736 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-03-09 22:49:18,320 INFO    MainThread:28736 [wandb_init.py:init():798] backend started and connected
2025-03-09 22:49:18,321 INFO    MainThread:28736 [wandb_init.py:init():891] updated telemetry
2025-03-09 22:49:18,321 INFO    MainThread:28736 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-09 22:49:18,981 INFO    MainThread:28736 [wandb_init.py:init():990] starting run threads in backend
2025-03-09 22:49:19,356 INFO    MainThread:28736 [wandb_run.py:_console_start():2375] atexit reg
2025-03-09 22:49:19,357 INFO    MainThread:28736 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-09 22:49:19,357 INFO    MainThread:28736 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-09 22:49:19,357 INFO    MainThread:28736 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-09 22:49:19,359 INFO    MainThread:28736 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-09 22:49:19,916 ERROR   MainThread:28736 [util.py:get_module():249] Error importing optional module PIL.Image
Traceback (most recent call last):
  File "C:\Users\singh\OneDrive\Documents\GitHub\DA6401_Assignment-01\.conda\Lib\site-packages\wandb\util.py", line 215, in import_module_lazy
    return sys.modules[name]
           ~~~~~~~~~~~^^^^^^
KeyError: 'PIL.Image'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\singh\OneDrive\Documents\GitHub\DA6401_Assignment-01\.conda\Lib\site-packages\wandb\util.py", line 244, in get_module
    return import_module_lazy(name)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\singh\OneDrive\Documents\GitHub\DA6401_Assignment-01\.conda\Lib\site-packages\wandb\util.py", line 217, in import_module_lazy
    spec = importlib.util.find_spec(name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib.util>", line 95, in find_spec
ModuleNotFoundError: No module named 'PIL'
2025-03-09 22:49:20,700 INFO    MsgRouterThr:28736 [mailbox.py:close():129] Closing mailbox, abandoning 1 handles.
